src/globalConfig.js

```diff
@@ -30,18 +33,19 @@ class GlobalConfigImpl {
         var config = configInput || {};
         this.stage = config.stage || this.stage;
         this.region = config.region || this.region;
+        this.cell = config.cell || this.cell;
         this.endpointOverride = config.endpoint || this.endpointOverride;
         this.reconnect = config.reconnect === false ? false : this.reconnect;
         this.messageReceiptThrottleTime = config.throttleTime ? config.throttleTime : 5000;
         this.features["values"] = Array.isArray(config.features) ? [...config.features] : new Array();
     }
 
-    getRegion() {
-        return this.region;
-    }
-
-    getEndpointOverride() {
-        return this.endpointOverride;
+    updateStageRegionCell(config) {
```

Comment:
Can you remove all the 'cell' related changes in this file and the unit test file? Cell concept is not supposed to be public. And I believe these are not used in public packages